pxtextension.cpp

```diff
@@ -294,6 +294,8 @@ namespace mlrunner {
         mlDataProcessor.deinit();
         free(actions);
         free(predictions);
+        actions = NULL;
+        predictions = NULL;
```

Comment:
This was just me noticing I forgot to reset the pointers to NULL in the `deinit()` function.